### PR TITLE
Sa 163 validate sidebar error input

### PR DESCRIPTION
**Pull request by sairaghav**
# Description

- Added reset button on sidebar error component that allows to reset the summary/knowledge graph (not implemented yet) after user enters incorrect paper id. **Requires design review and design changes.**: https://smarticle.atlassian.net/wiki/spaces/SA/pages/6619407/Requirements+Changelog
- Disabled Submit button when there is no input provided
- Changed the footer in the sidebar for Semantic Scholar to display as a link
- Added tests for summary: https://smarticle.atlassian.net/wiki/spaces/SA/pages/13205506/Testing+Documentation

## Type of change

Please delete options that are not relevant.

- New feature (non-breaking change which adds functionality)
- This change requires a documentation update

## Screenshots
**Initial State**
image
**After providing custom paper ID**
image
**After Reset**
image

# Reference JIRA tickets : 

- SA-163

# Checklist:

Put an "x" in the brackets to check the checkbox.

- [x ] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [ x] I have commented my code in particularly hard-to-understand areas
- [ x] I have made corresponding changes to the documentation
- [ x] I have made corresponding changes to JIRA tickets
- [ x] My changes generate no new warnings
- [ x] I have added tests that prove my fix is effective or that my feature works
- [ x] New and existing unit tests pass locally with my changes
- [ x] I have merged develop into my branch and resolved possible conflicts

### CODE REVIEW

File: src/components/SidebarError/SidebarError.scss [hunk 1]
```diff
@@ -60,9 +60,14 @@
           color: var(--sa-neutral-color-3);
         }
       }
-      .sidebar-error-button {
+      .sidebar-error-buttons{
+        display: flex;
         text-align: center;
-        margin-bottom: 13px;
+        margin-bottom: 16px;
+        justify-content: flex-start;
```
Comment (nMedic246) [inline src/components/SidebarError/SidebarError.scss]: I would change justify-content to center, it looks a bit nicer. 
(idk i just like things to be symmetrical, but if you like it this way just leave it as it is :) )

**Review by nMedic246 — COMMENTED** [review]
It looks nice! Also the test are thorough and they all work fine. 
The only thing that it would be nice to change is to remove the user input in the text field after the reset button is clicked.

File: src/components/SidebarError/SidebarError.scss (hunk 1 above)
Comment (sairaghav) [reply src/components/SidebarError/SidebarError.scss]: Made the changes. It was supposed to be centered according to the design, but it looked weird when the sidebar was resized to the max.

Comment (sairaghav) [reply src/components/SidebarError/SidebarError.scss]: Also cleared the text field input on reset

**Review by nMedic246 — APPROVED** [review]
Good job!

**Pull request merged**